plugins/boxlite-agent-tooling/host-parity.test.sh

```
@@ -292,8 +298,47 @@ for pair in "claude:$CLAUDE_HOOKS:asyncRewake" "codex:$CODEX_HOOKS:async"; do
-claude_normalized="$(jq -Sc '
-  .hooks.SubagentStart[].hooks[] |=
+# Codex's hook-event set is closed (HookEventsToml in the shipped binary). Held here as
+# data so a newly wired event is checked against it without needing a Codex install:
+# getting this wrong does not disable one hook, it disables all of them.
+CODEX_EVENTS='PreToolUse PermissionRequest PostToolUse PreCompact PostCompact
+SessionStart SessionEnd UserPromptSubmit SubagentStart SubagentStop Stop Interrupt'
+# Events Claude Code has and Codex does not. Wired only in hooks/hooks.json.
+CLAUDE_ONLY_EVENTS='StopFailure'
+
+for event in $CLAUDE_ONLY_EVENTS; do
+  case " $(echo $CODEX_EVENTS) " in
```

Shellcheck (0.11.0)

[style] Useless echo? Instead of 'cmd $(echo foo)', just use 'cmd foo'.

(SC2116)